src/components/Forest/Forest.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+import FarmhandContext from '../Farmhand/Farmhand.context'
+
+export const Forest = () => {
+  return <div>'welcome to da forest'</div>
+}
+
+export default function Consumer(props) {
+  return (
+    <FarmhandContext.Consumer>
+      {({ gameState, handlers }) => (
+        <Forest {...{ ...gameState, ...handlers, ...props }} />
+      )}
+    </FarmhandContext.Consumer>
+  )
+}
```

Comment:
This pattern will certainly work, and it's what most of the components in the codebase still use. Recently, I've been getting into the habit of using the [`useContext` hook](https://react.dev/reference/react/useContext) as it's a bit simpler and more modern.

Here's an example of it in use: https://github.com/jeremyckahn/farmhand/blob/37f60eab64fb7fbdfcfe4dbad5244ef914cc23f9/src/components/SettingsView/RandomSeedInput.js#L10-L13

Notice that it doesn't have a `Consumer` component at the bottom of the file that most of the older components have.

Here's how it can be set up for testing: https://github.com/jeremyckahn/farmhand/blob/37f60eab64fb7fbdfcfe4dbad5244ef914cc23f9/src/components/SettingsView/RandomSeedInput.test.js#L11-L17

Both approaches are valid, I just want to call out this alternative in case you like it better. 🙂

Reply:
ahh yes, good call on moving to `useContext`. i actually hadn't noticed that in this repo yet, but i'm all for moving to it!